### DISCUSSION TITLE
Control states in `Accordion` component loses transition?

### DISCUSSION THREAD

**Discussion by jhdcruz** [mantinedev/mantine#6725] [Q&A] [answered]
Using control states in `<Accordion>` loses the transition and becomes instant.

I don't know if this is a bug or just a problem on my side, so I asked this in Q&A instead.

Sample Code:
```tsx
 // ...
 const [value, setValue] = useState<string[]>([]);

 // ommited...

 <Accordion
      multiple={true}
      onChange={setValue}
      transitionDuration={200} // with or without doesn't do anything.
      value={value}
    >
	{/* ommited... */}
</Accordion>
```

**Comment by rtivital** [comment, marked as answer]
I cannot reproduce the issue – https://codesandbox.io/p/sandbox/mantine-react-template-forked-5h5p6c?file=%2Fsrc%2FApp.tsx%3A27%2C23&workspaceId=dd881b63-29f6-464c-bcdf-b5f78ad65b26

**Reply by jhdcruz** [reply]
Seems to be a problem on my side...
